docs/source/howto/plugins_develop.rst

```diff
@@ -18,26 +18,24 @@ Creating a plugin package
 =========================
 
 
-AiiDA plugins can be bundled and distributed in a `Python package <packages>`_ that provides a set of extensions to AiiDA.
+AiiDA plugins can be bundled and distributed in a `Python package <https://docs.python.org/3/tutorial/modules.html?highlight=package#packages>`_ that provides a set of extensions to AiiDA.
 
 .. note::
 
   The Python community uses the term 'package' rather loosely.
-  Depending on context, it may refer simply to a folder containing individual Python modules or it may include the files necessary for building and installing a package to be distributed via the `Python Package Index (PyPI) <pypi>`_.
-
-.. _packages: https://docs.python.org/2/tutorial/modules.html?highlight=package#packages
-
+  Depending on context, it may refer simply to a folder containing individual Python modules or it may include the files necessary for building and installing a package to be distributed via the `Python Package Index (PyPI) <https://pypi.python.org>`_.
 
 Quickstart
 ----------
 
-The fastest way to jumpstart an AiiDA plugin package is to use the `AiiDA plugin cutter <plugin-cutter>`_ in order to template the basic folder structure, already customized according to the desired name of your plugin, following AiiDA conventions.
+The fastest way to jumpstart an AiiDA plugin package is to use the AiiDA `plugin cutter`_ in order to template the basic folder structure, already customized according to the desired name of your plugin, following AiiDA conventions.
 
-Simply go to the `AiiDA plugin cutter <plugin-cutter>`_ and follow the usage instructions.
+Simply go to the AiiDA `plugin cutter`_ and follow the usage instructions.
 See also the `aiida-diff`_ demo plugin package for an in-depth explanation of the files & folders produced by the plugin cutter.
 
 In the following, we explain some of the conventions implemented by the AiiDA plugin cutter.
 
+.. _plugin cutter: https://github.com/aiidateam/aiida-plugin-cutter
```

Comment:
I think it is better if all the link definitions are at the bottom of the document, as they were previously